jadx-gui/src/main/resources/i18n/Messages_de_DE.properties

```diff
@@ -128,28 +128,28 @@ preferences.title=Einstellungen
 preferences.deobfuscation=Deobfuskierung
 preferences.appearance=Aussehen
 preferences.decompile=Dekompilierung
-#preferences.plugins=Plugins
+preferences.plugins=Plugins
 preferences.project=Projekt
 preferences.other=Andere
 preferences.language=Sprache
 preferences.lineNumbersMode=Editor Zeilennummern-Modus
-#preferences.jumpOnDoubleClick=Enable jump on double click
+preferences.jumpOnDoubleClick=Sprung bei Doppelklick aktivieren
 preferences.check_for_updates=Nach Updates beim Start suchen
-#preferences.useDx=Use dx/d8 to convert java bytecode
-#preferences.decompilationMode=Decompilation mode
-#preferences.codeCacheMode=Code cache mode
+preferences.useDx=dx/d8 zur Konvertierung von Java Bytecode verwenden
+preferences.decompilationMode=Dekompilierungsmodus
+preferences.codeCacheMode=Cache-Code-Modus
 preferences.showInconsistentCode=Inkonsistenten Code anzeigen
 preferences.escapeUnicode=Unicodezeichen escapen
 preferences.replaceConsts=Konstanten ersetzen
 preferences.respectBytecodeAccessModifiers=Modifikatoren für Bytecode-Zugriff beachten
 preferences.useImports=Import statements generieren
-#preferences.useDebugInfo=Use debug info
+preferences.useDebugInfo=Debug-Infos verwenden
 preferences.inlineAnonymous=Anonyme Inline-Klassen
 preferences.inlineMethods=Inline-Methoden
 preferences.fsCaseSensitive=Dateisystem unterscheidet zwischen Groß/Kleinschreibung
 preferences.skipResourcesDecode=Keine Ressourcen dekodieren
-#preferences.useKotlinMethodsForVarNames=Use kotlin methods for variables rename
-#preferences.commentsLevel=Code comments level
+preferences.useKotlinMethodsForVarNames=Kotlin-Methoden für die Umbenennung von Variablen verwenden
+preferences.commentsLevel=Code-Kommentarebene
```

Comment:
If I remember correctly this preferences entry manages how verbose the code comments are that are included into the decompiled Java code. I don't think someone will understand this by `Code-Kammentarebene`. What do you think of `Level der Code Kommentierungen`?